pallets/multisig/src/benchmarking.rs

```diff
@@ -0,0 +1,443 @@
+        let multisig = generate_multisig::<T>(alice.account(), alice.origin(), signers)?;
+        let proposal_id = <MultiSig<T>>::ms_tx_done(multisig.clone());
+        <MultiSig<T>>::create_proposal_as_identity(
+            alice.origin().into(),
+            multisig.clone(),
+            Box::new(frame_system::Call::<T>::remark(vec![]).into()),
+            Some(<Timestamp<T>>::get() + 1.into()),
+            true
+        )?;
+    }: _(bob.origin(), multisig.clone(), 0)
+    verify {
+        ensure!(
+            <MultiSig<T>>::votes((multisig.clone(), Signatory::from(bob.did()), proposal_id)),
+            "reject_as_identity"
+        );
+    }
+
+
+    reject_as_key {
+        let alice = <UserBuilder<T>>::default().generate_did().build("alice");
+        let bob = <UserBuilder<T>>::default().build("bob",);
+        let charlie = <UserBuilder<T>>::default().build("charlie",);
+        let signers = vec![
+            Signatory::Account(bob.account()),
+            Signatory::Account(charlie.account()),
+        ];
+        let multisig = generate_multisig::<T>(alice.account(), alice.origin(), signers)?;
+        let proposal_id = <MultiSig<T>>::ms_tx_done(multisig.clone());
+        <MultiSig<T>>::create_proposal_as_key(
+            bob.origin().into(),
+            multisig.clone(),
+            Box::new(frame_system::Call::<T>::remark(vec![]).into()),
+            Some(<Timestamp<T>>::get() + 1.into()),
+            true
+        )?;
+    }: _(charlie.origin(), multisig.clone(), 0)
+    verify {
+        ensure!(
+            <MultiSig<T>>::votes((multisig.clone(), Signatory::Account(charlie.account()), proposal_id)),
+            "reject_as_key"
+        );
+    }
+
+    accept_multisig_signer_as_identity {
+        let alice = <UserBuilder<T>>::default().generate_did().build("alice");
+        let multisig = <MultiSig<T>>::get_next_multisig_address(alice.account());
+        <MultiSig<T>>::create_multisig(alice.origin().into(), vec![Signatory::from(alice.did())], 1)?;
+        let alice_auth_id = get_last_auth_id::<T>(&Signatory::from(alice.did()));
+        Context::set_current_identity::<Identity<T>>(Some(alice.did()));
+        let num_of_signers = <NumberOfSigners<T>>::get(multisig.clone());
+    }: _(alice.origin(), alice_auth_id)
+    verify {
+        ensure!(
+            num_of_signers < <NumberOfSigners<T>>::get(multisig.clone()),
+            "accept_multisig_signer_as_identity"
+        );
+    }
+
+    accept_multisig_signer_as_key {
+        let alice = <UserBuilder<T>>::default().generate_did().build("alice");
+        let bob = <UserBuilder<T>>::default().build("bob",);
+        let multisig = <MultiSig<T>>::get_next_multisig_address(alice.account());
+        <MultiSig<T>>::create_multisig(alice.origin().into(), vec![Signatory::Account(bob.account())], 1)?;
+        let auth_id = get_last_auth_id::<T>(&Signatory::Account(bob.account()));
+        let num_of_signers = <NumberOfSigners<T>>::get(multisig.clone());
```

Comment:
Setup is near identical to the one above save for alice vs bob; let's refactor into a function.